packages/harmony/src/icons/IconDetails.stories.mdx

```diff
@@ -0,0 +1,145 @@
+import { Meta, Title } from '@storybook/blocks'
+
+import { Flex } from 'components'
+import { IconQuestionCircle, IconStars } from 'icons'
+import addIconArrangement from 'storybook/assets/addIconArrangement.png'
+import exportIconCommand from 'storybook/assets/exportIconCommand.png'
+import exportIconLayer from 'storybook/assets/exportIconLayer.png'
+import verifyIconSettings from 'storybook/assets/verifyIconSettings.png'
+
+<Meta title='Icons/Details' />
+
+<Title>Details</Title>
+
+### Overview
+
+An icon is a graphic symbol designed to visually indicate the purpose of an interface element.
+
+export function Step(props) {
+  const { index, children } = props
+  return (
+    <section
+      css={(theme) => ({
+        background: theme.color.background.surface1,
+        borderRadius: theme.cornerRadius.s,
+        border: `1px solid ${theme.color.border.strong}`,
+        padding: theme.spacing.l,
+        marginTop: theme.spacing['2xl']
+      })}
+    >
+      {index === undefined ? null : <h2>Step {index}</h2>}
+      {children}
+    </section>
+  )
+}
+
+# For Developers
+
+export const headerIconCss = {
+  visibility: 'visible !important',
+  top: '3px !important'
+}
+
+## <IconQuestionCircle color='default' css={headerIconCss} /> How to Export Icons
+
+Follow the general guide below to learn how to export icons from the library
+
+<Step index={1}>
+
+Find the icon you want to export.
+Pro Tip: You can search for icons by pressing Cmd + F
```

Comment:
I don't think it's obvious I'm supposed to be in Figma at this point. Can we start with a link?